metadata_fetcher/fetchers/Fetcher.py

```diff
@@ -44,11 +44,22 @@ def __init__(self, params: dict):
         self.collection_id = params.get('collection_id')
         self.write_page = params.get('write_page', 0)
         self.vernacular_version = params['vernacular_version']
-
+        self.http_session = self.configure_http_session()
 
         if not self.collection_id:
             raise CollectionIdRequired("collection_id is required")
 
+    def configure_http_session(self) -> requests.Session:
+        http = requests.Session()
+        retry_strategy = Retry(
+            total=3,
+            status_forcelist=[413, 429, 500, 502, 503, 504],
```

Comment:
Why did you remove the back off factor - that seems useful and relevant?

Reply:
I also wonder if this makes sense as a global configuration - like in `rikolti/utils.py`?

Reply:
Well, I didn't actually remove the backoff factor as there wasn't one configured at all here previously. I figured that we could go with the default backoff factor of 0 and tweak if necessary. But sure, it probably makes sense to set it to `2`.

Yeah, we could make this a global configuration. This is actually a copy of `configure_http_session()` from the content harvester code (which doesn't have a backoff factor set).